src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -98,27 +118,61 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewIni
     return this._isSelected;
   }
 
+  /**
+   * Indicates whether users can change the order of the repeater item.
+   * The repeater component's reorderable property must also be set to `true`.
+   */
   @Input()
-  public selectable: boolean = false;
+  public reorderable: boolean = false;
 
+  /**
+   * Indicates whether to display a checkbox in the left of the repeater item.
+   */
   @Input()
-  public reorderable: boolean = false;
+  public selectable: boolean = false;
 
+  /**
+   * Indicates whether to display an inline form within the repeater.
+   * Users can toggle between displaying and hiding the inline form.
+   */
   @Input()
   public showInlineForm: boolean = false;
 
+  /**
+   * Specifies an object that the repeater component returns for this repeater item
+   * when the `orderChange` event fires. Required if you set the `reorderable` property to `true`.
+   */
+  @Input()
+  public tag: any;
+
+  /**
+   * Fires when users collapse the repeater item.
+   */
   @Output()
   public collapse = new EventEmitter<void>();
 
+  /**
+   * Fires when users expand the repeater item.
+   */
   @Output()
   public expand = new EventEmitter<void>();
 
+  /**
+   * Fires when the repeater includes an inline form and users close it. This event emits
+   * [a SkyInlineFormCloseArgs type](https://developer.blackbaud.com/skyux/components/inline-form#skyinlineformcloseargs-properties).
```

Comment:
Should `SkyInlineFormCloseArgs` be a code snippet? I think so.

Reply:
Yep. Done.